From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: conda-forge/recipe <conda@example.com>
Date: Wed, 26 Mar 2025 12:00:00 +0000
Subject: [PATCH] cdbpp: drop std::binary_function (removed in C++17)

libc++ with Clang 20 no longer provides std::binary_function; the
murmurhash2 functor does not need the empty base typedefs it supplied.
---
 pysimstring/simstring/cdbpp.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/pysimstring/simstring/cdbpp.h b/pysimstring/simstring/cdbpp.h
index 1111111..2222222 100644
--- a/pysimstring/simstring/cdbpp.h
+++ b/pysimstring/simstring/cdbpp.h
@@ -77,8 +77,7 @@ enum {
  *  @author Austin Appleby
  */
-class murmurhash2 :
-    public std::binary_function<const void *, size_t, uint32_t>
+class murmurhash2
 {
 protected:
     inline static uint32_t get32bits(const char *d)
     {
